zwave_js_server/client.py

```diff
@@ -40,7 +37,6 @@ def __init__(self, ws_server_url: str, aiohttp_session: ClientSession):
         # The WebSocket client
         self._client: Optional[ClientWebSocketResponse] = None
         # Current state of the connection
-        self.state = STATE_DISCONNECTED
```

Comment:
This is a breaking change. We need to remember to update the websocket api in the core for this.

Reply:
We should be able to easily migrate to using `connected`